packages/backend-function/src/index.internal.ts

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line @typescript-eslint/naming-convention
+import * as __export__runtime from './runtime/index.js';
+// eslint-disable-next-line @typescript-eslint/naming-convention
+import * as __export__get_amplify_clients_configuration from './runtime/get_amplify_clients_configuration.js';
+
+/*
+ Api-extractor does not ([yet](https://github.com/microsoft/rushstack/issues/1596)) support multiple package entry points
+ Because this package has a submodule export, we are working around this issue by including that export here and directing api-extract to this entry point instead
+ This allows api-extractor to pick up the submodule exports in its analysis
+ */
+export { __export__runtime, __export__get_amplify_clients_configuration };
```

Comment:
```suggestion
// eslint-disable-next-line @typescript-eslint/naming-convention
import * as __export__runtime from './runtime/index.js';

/*
 Api-extractor does not ([yet](https://github.com/microsoft/rushstack/issues/1596)) support multiple package entry points
 Because this package has a submodule export, we are working around this issue by including that export here and directing api-extract to this entry point instead
 This allows api-extractor to pick up the submodule exports in its analysis
 */
export { __export__runtime };
```